core/src/mindustry/logic/LAssembler.java

```diff
@@ -108,6 +115,14 @@ public LVar var(String symbol){
         return Color.toDoubleBits(r, g, b, a);
     }
 
+    double parseNamedColor(String symbol){
+        if (symbol.equals("%[stat]") || symbol.equals("%[accent]")) return Pal.stat.toDoubleBits();
```

Comment:
Why is this a separate case?

Reply:
I couldn't find anything about Colors.stat or Colors.accent, where is it?

Reply:
It is already registered in `Colors`, check `UI.java`. This code is unnecessary.

Reply:
thanks
also learned about new named colors that were not mentioned in #help pins
should be fixed in https://github.com/Anuken/Mindustry/pull/10682/commits/987c9c30a3c1726826ee9b5a34212c4483cc10f2

Reply:
thanks
also learned about new named colors that were not mentioned in #help pins
should be fixed in https://github.com/Anuken/Mindustry/pull/10682/commits/987c9c30a3c1726826ee9b5a34212c4483cc10f2